DigitalLearningSolutions.Web/ViewModels/Signposting/LinkLearningHubSso/LinkLearningHubViewModel.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.Signposting.LinkLearningHubSso
+{
+    public class LinkLearningHubViewModel
+    {
+        public LinkLearningHubViewModel(bool isAccountAlreadyLinked, int? learningHubResourceId)
+        {
+            ShowIsAlreadyLinkedWarning = isAccountAlreadyLinked;
+            ResourceLinkId = learningHubResourceId;
+            ShowResourceLink = learningHubResourceId.HasValue;
+        }
+
+        public bool ShowIsAlreadyLinkedWarning { get; set; }
+
+        public int? ResourceLinkId { get; set; }
+
+        public bool ShowResourceLink { get; set; }

```

Comment:
This could be slightly clearer as `=> ResourceLinkId.HasValue`

Reply:
Done.